ethcore/sync/src/chain/handler.rs

```diff
@@ -745,9 +745,7 @@ mod tests {
 
 		let block = Rlp::new(&block_data);
 
-		let result = SyncHandler::on_peer_new_block(&mut sync, &mut io, 0, &block);
-
-		assert!(result.is_ok());
+		SyncHandler::on_peer_new_block(&mut sync, &mut io, 0, &block).expect("result to be ok");
```

Comment:
I think I'd prefer using an assert here like before: it's easier to spot what the test is doing if I can scan the code and look for the `assert`. Just a personal preference, but yeah, please consider it.

Reply:
the problem with `assert!(result.is_ok()` is that it gives no information what went wrong. It will report only `expected true to be equal false`, whereas `expect` will use `fmt::Debug` to print unwrapped error

Reply:
Gotcha, didn't think about that. Fine by me then.